internal/terraform/lang/provider_block.go

```diff
@@ -70,7 +70,9 @@ func (p *providerBlock) Labels() []*ParsedLabel {
 	if p.labels != nil {
 		return p.labels
 	}
-	p.labels = parseLabels(p.BlockType(), p.labelSchema, p.hclBlock.Labels)
+	labels, _ := ParseLabels(p.tokens, p.labelSchema)
```

Comment:
Same as above, could you add a comment here?

Reply:
Given that `ParseLabels` doesn't actually need to return any error anymore I will just remove the err from the return values.